src/KickMCMC.jl

```diff
@@ -1,8 +1,17 @@
 using Turing
 using Distributions
 
+########################
+### 
+### Struct definitions
+### 
+########################
+
 """
-RTW TODO: figure out correct way to document structs
+    struct WrappedCauchy{T1<:Real, T2<:Real} <: ContinuousUnivariateDistribution
+
+The WrappedCauchy distribution captures the Cauchy distribution defined on the unit 
```

Comment:
maybe instead of capture, say resemble. The wrapped cauchy is similar to the cauchy distribution near the mean, but wraps around at the edges